trasnlators.sh not working

`./translators.sh` is not working since pulled the latest. (Before pulling, it was working.)

the output of the terminal is below:
```
$ ./translators.sh 
Temporary file: /tmp/tmp.5oWVnKpuLSBAZAAR
Setting im_a_translator to true
Directory already configured.

Just run your build command (e.g. ninja) and Meson will regenerate as necessary.
Run "meson setup --reconfigure" to force Meson to regenerate.

If build failures persist, run "meson setup --wipe" to rebuild from scratch
using the same options as passed when configuring the build.

Number of languages currently in file 22
Language codes currently in LINGUAS file.
# Please keep this file sorted alphabetically.
ar
bg
bs
cs
de
es
fa_IR
fr
hu
id
it
ja
nl
pl
pt_BR
pt_PT
ru
tr
uk
zh_CN
zh_TW
Proceed? (Type y or Y to confirm)y
Proceeding...

Copying LINGUAS to /tmp/LINGUAS.backup
'LINGUAS' -> '/tmp/LINGUAS.backup'
Type the language code you want to enable translation for. ex.: pt_BR or es: ja
Found ja in the system's language code list.
ja
ja already in file
# Please keep this file sorted alphabetically.
ar
bg
bs
cs
de
es
fa_IR
fr
hu
id
it
ja
nl
pl
pt_BR
pt_PT
ru
tr
uk
zh_CN
zh_TW
Number of languages currently in file 22
# Please keep this file sorted alphabetically.
ar
bg
bs
cs
de
es
fa_IR
fr
hu
id
it
ja
nl
pl
pt_BR
pt_PT
ru
tr
uk
zh_CN
zh_TW
# Please keep this file sorted alphabetically.
ar
bg
bs
cs
de
es
fa_IR
fr
hu
id
it
ja
nl
pl
pt_BR
pt_PT
ru
tr
uk
zh_CN
zh_TW
~/GitWork/bazzar
~/GitWork/bazzar/build ~/GitWork/bazzar
Generating the main pot (Portable Object Template) file for lang ja...
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /home/su6c4t/.local/bin/ninja bazaar-pot
[0/1] Regenerating build files
The Meson build system
Version: 1.9.0
Source dir: /home/su6c4t/GitWork/bazzar
Build dir: /home/su6c4t/GitWork/bazzar/build
Build type: native build
WARNING: You should add the boolean check kwarg to the run_command call.
         It currently defaults to false,
         but it will default to true in meson 2.0.
         See also: https://github.com/mesonbuild/meson/issues/9300
Project name: bazaar
Project version: 0.5.10 (vcs=ebedd17)
C compiler for the host machine: cc (gcc 11.4.0 "cc (Ubuntu 11.4.0-1ubuntu1~22.04.2) 11.4.0")
C linker for the host machine: cc ld.bfd 2.38
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program msgfmt found: YES (/usr/bin/msgfmt)
Program msginit found: YES (/usr/bin/msginit)
Program msgmerge found: YES (/usr/bin/msgmerge)
Program xgettext found: YES (/usr/bin/xgettext)
Build targets in project: 24

bazaar 0.5.10 (vcs=ebedd17)

  User defined options
    im_a_translator: true

Found ninja-1.13.0.git.kitware.jobserver-pipe-1 at /home/su6c4t/.local/bin/ninja
Cleaning... 0 files.
[0/2] Running external command bazaar-pot
/usr/bin/xgettext: warning: file 'src/bz-addons-dialog.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-app-tile.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-app-size-dialog.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-browse-widget.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-decorated-screenshot.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-detailed-app-tile.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-entry-inspector.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-featured-tile.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-flathub-page.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-full-view.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-hardware-support-dialog.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-inspector.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: warning: file 'src/bz-installed-page.blp' extension 'blp' is unknown; will try C
/usr/bin/xgettext: error while opening "src/bz-issue.c" for reading: No such file or directory
FAILED: [code=1] meson-internal__bazaar-pot 
/home/su6c4t/.local/bin/meson --internal gettext pot --pkgname=bazaar --source-root=/home/su6c4t/GitWork/bazzar/ --subdir=po --extra-args=--from-code=UTF-8@@--add-comments@@--keyword=_@@--keyword=N_@@--keyword=C_:1c,2@@--keyword=NC_:1c,2@@--keyword=g_dcgettext:2@@--keyword=g_dngettext:2,3@@--keyword=g_dpgettext2:2c,3@@--flag=N_:1:pass-c-format@@--flag=C_:2:pass-c-format@@--flag=NC_:2:pass-c-format@@--flag=g_dngettext:2:pass-c-format@@--flag=g_strdup_printf:1:c-format@@--flag=g_string_printf:2:c-format@@--flag=g_string_append_printf:2:c-format@@--flag=g_error_new:3:c-format@@--flag=g_set_error:4:c-format@@--flag=g_markup_printf_escaped:1:c-format@@--flag=g_log:3:c-format@@--flag=g_print:1:c-format@@--flag=g_printerr:1:c-format@@--flag=g_printf:1:c-format@@--flag=g_fprintf:2:c-format@@--flag=g_sprintf:2:c-format@@--flag=g_snprintf:3:c-format --xgettext=/usr/bin/xgettext
ninja: build stopped: subcommand failed.
Update and/or create the po (Portable Object) files for ja.
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /home/su6c4t/.local/bin/ninja bazaar-update-po
[0/1] Running external command bazaar-update-po
xgettext: warning: file 'src/bz-addons-dialog.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-app-tile.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-app-size-dialog.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-browse-widget.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-decorated-screenshot.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-detailed-app-tile.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-entry-inspector.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-featured-tile.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-flathub-page.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-full-view.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-hardware-support-dialog.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-inspector.blp' extension 'blp' is unknown; will try C
xgettext: warning: file 'src/bz-installed-page.blp' extension 'blp' is unknown; will try C
xgettext: error while opening "src/bz-issue.c" for reading: No such file or directory
FAILED: [code=1] meson-internal__bazaar-update-po 
/home/su6c4t/.local/bin/meson --internal gettext update_po --pkgname=bazaar --source-root=/home/su6c4t/GitWork/bazzar/ --subdir=po --extra-args=--from-code=UTF-8@@--add-comments@@--keyword=_@@--keyword=N_@@--keyword=C_:1c,2@@--keyword=NC_:1c,2@@--keyword=g_dcgettext:2@@--keyword=g_dngettext:2,3@@--keyword=g_dpgettext2:2c,3@@--flag=N_:1:pass-c-format@@--flag=C_:2:pass-c-format@@--flag=NC_:2:pass-c-format@@--flag=g_dngettext:2:pass-c-format@@--flag=g_strdup_printf:1:c-format@@--flag=g_string_printf:2:c-format@@--flag=g_string_append_printf:2:c-format@@--flag=g_error_new:3:c-format@@--flag=g_set_error:4:c-format@@--flag=g_markup_printf_escaped:1:c-format@@--flag=g_log:3:c-format@@--flag=g_print:1:c-format@@--flag=g_printerr:1:c-format@@--flag=g_printf:1:c-format@@--flag=g_fprintf:2:c-format@@--flag=g_sprintf:2:c-format@@--flag=g_snprintf:3:c-format --msginit=/usr/bin/msginit --msgmerge=/usr/bin/msgmerge
ninja: build stopped: subcommand failed.
--------------------------------

Configuration done. Now ready for you to open your "po" file in your text editor and begin translating.
When you are done, commit your changes form your fork and submit a pull request on 
https://github.com/kolunmi/bazaar/blob/master/TRANSLATORS.md also, refer to TRANSLATORS.md if needed 

--------------------------------

```

--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -44,7 +44,6 @@ src/bz-flatpak-entry.c
 src/bz-flatpak-instance.c
 src/bz-full-view.blp
 src/bz-full-view.c
-src/bz-global-progress.c
 src/bz-global-state.c
 src/bz-gnome-shell-search-provider.c
 src/bz-group-tile-css-watcher.c
@@ -63,12 +62,10 @@ src/bz-preferences-dialog.blp
 src/bz-preferences-dialog.c
 src/bz-progress-bar.blp
 src/bz-progress-bar.c
-src/bz-release.c
 src/bz-releases-dialog.blp
 src/bz-releases-list.blp
 src/bz-releases-list.c
 src/bz-result.c
-src/bz-review.c
 src/bz-rich-app-tile.blp
 src/bz-rich-app-tile.c
 src/bz-screenshot.c

